src/dags/importscripts/import_risicozones.py

```diff
@@ -0,0 +1,393 @@
+import ccsv as csv
+import json
+import sys
+import hashlib
+from collections import namedtuple
+from collections import defaultdict
+from shapely import wkt
+from shapely.geometry import (
+    GeometryCollection,
+    MultiLineString,
+    LineString,
+    MultiPolygon,
+    Polygon,
+)
+
+
+# set max number of karakters per cell
+csv.field_size_limit(sys.maxsize)
+
+# ----------------------------#
+# helper functions
+# ----------------------------#
+
+
+def get_dialect(file: str):
+    """determine dialect of file"""
+    with open(file) as data:
+        return csv.Sniffer().sniff(data.readline())
+
+
+def get_header(file: str, extra_fields_to_add: list = None):
+    """retreive header from file"""
+    with open(file) as data:
+        reader = csv.reader(data, dialect=get_dialect(file))
+        # reader = csv.reader(data, delimiter="|")
+        header = list(map(lambda s: s.lower(), next(reader)))
+        if extra_fields_to_add:
+            for col in extra_fields_to_add:
+                header.append(col)
+        return header
+
+
+def create_named_tuple(file: str, class_name: str, header: list = None):
+    """create named tuple based upon header file"""
+    if not header:
+        header = get_header(file)
+    return namedtuple(class_name, header)
+
+
+def file_parser(file: str):
+    """generate data from file"""
+    with open(file) as data:
+        yield from csv.reader(data, dialect=get_dialect(file))
+
+
+def file_iter(file: str, name_named_tuple: str, header: list = None):
+    """retrieve data of file into named tuple"""
+    named_tuple = create_named_tuple(file, name_named_tuple, header)
+    reader = file_parser(file)
+    next(reader)  # skip header, just get data
+    for row in reader:
+        yield named_tuple(*row)
+
+
+def filter_rows(row, filter: dict = None):
+    """return true on rows that meet criterium"""
+    FILTER_OPTIONS = {
```

Comment:
Je zou hier ook de `operator` module voor kunnen gebruiken. Dan heb je geen `eval()` nodig. Want, `eval` is evil :-) Theoretisch zou iemand een getruct csv bestand kunnen aanleveren met arbitrary python code erin.

Reply:
Eens, eval kan gevaarlijk zijn. Ik heb het verwijderd en via operator laten doen :+1: